### PR TITLE
Generate sample people data

### PR DESCRIPTION
**Pull request by elayabharath**

### CODE REVIEW

File: data_tools/README.md [hunk 1]
```diff
@@ -3,4 +3,4 @@ Firebase console > project settings > service accounts
 Add the Firebase Service Account secret as `serviceAccount.json` into the current folder
```
Comment (lukechurch) [inline data_tools/README.md]: I think this is now out of date as we have a path - consider removing.

File: data_tools/README.md [hunk 2]
```diff
@@ -3,4 +3,4 @@ Firebase console > project settings > service accounts
...
 
 # Write sample
-`python3 writeSample.py </path/to/fb_secret.json> </path/to/fb_const.json> <themes|filters>`
+`python3 writeSample.py </path/to/fb_secret.json> </path/to/fb_const.json> <themes|filters|people>`
```
Comment (lukechurch) [inline data_tools/README.md]: I think it's a fb_service_account.json isn't it, rather than a secret?

**Review by lukechurch — APPROVED** [review]

File: data_tools/README.md (hunk 1 above)
Comment (elayabharath) [reply data_tools/README.md]: Ah yes fixed!

File: data_tools/README.md (hunk 2 above)
Comment (elayabharath) [reply data_tools/README.md]: Fixed

**Pull request merged**